### ISSUE TITLE
HTML element missing lang attribute

### ISSUE THREAD

**Issue by Sukha11y** [Sukha11y/Test-ZT-fix#2]
Problem
Where is it located
index.html - [https:///c%3A/Users/SukhadaPawar/Test-ZT-fix/Test-ZT-fix/index.html](https://c/Users/SukhadaPawar/Test-ZT-fix/Test-ZT-fix/index.html)

Locate element
XPath or CSS selector
html

Copy selector to clipboard
HTML element
<html style="">...</html>

Copy HTML element to clipboard
Description
The root HTML element does not have a lang attribute set.
Why it matters
When there is no language set on the html element, screen readers may not interpret the page in its intended language.

Solution
What to do
Ensure that the lang attribute is set within the html element.
Always use a language attribute on the HTML element. This is inherited by all other elements, so it will set a default language for the text in the document head element.

Note that you need to specify the language on the HTML element rather than the body element, since the body element doesn't cover the text inside the document's head element.
